### PR TITLE
Add a step for reseeding the cluster after migration

### PR DESCRIPTION
**Pull request by renetapopova**

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1488.surge.sh

Updated pages:
- [Store formats](https://neo4j-docs-operations-1488.surge.sh/operations-manual/5/database-internals/store-formats)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1488.surge.sh

Updated pages:
- [Store formats](https://neo4j-docs-operations-1488.surge.sh/operations-manual/5/database-internals/store-formats)

**Review by neo-ionut — CHANGES_REQUESTED** [review]

File: modules/ROOT/pages/database-internals/store-formats.adoc [hunk 1]
```diff
@@ -83,6 +85,8 @@ For example:
 ----
 neo4j-admin database copy --to-format="block" mydb blockdb
 ----
+. If you are working with a cluster, re-seed the database after migration.
```
[the remark this answers is not in the document]
Comment (renetapopova) [reply modules/ROOT/pages/database-internals/store-formats.adoc:88]: I'll update the steps.

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1488.surge.sh

Updated pages:
- [Store formats](https://neo4j-docs-operations-1488.surge.sh/operations-manual/5/database-internals/store-formats)

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-operations-1488.surge.sh

Updated pages:
- [Store formats](https://neo4j-docs-operations-1488.surge.sh/operations-manual/5/database-internals/store-formats)

**Review by neo-ionut — APPROVED** [review]
Very nice!

**Pull request merged**